Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -134,6 +134,25 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
                     bool mergeIssue = false;
                     bool notifyReplaceAPI = false;
                     AddAPIModelWizard.LearnedAPIModelsList.Clear();
+                    List<DeltaAPIModel> repeatingMatchingModels = AddAPIModelWizard.DeltaModelsList.Where(m => m.SelectedOperationEnum == DeltaAPIModel.eHandlingOperations.ReplaceExisting ||
+                                                                m.SelectedOperationEnum == DeltaAPIModel.eHandlingOperations.MergeChanges)
+                                                                        .GroupBy(m => m.matchingAPIModel).SelectMany(m => m.Skip(1)).ToList();
+
+                    bool matchingModelsReplacementIssue = repeatingMatchingModels.Count() != 0;
+                    int apiCount = 0;
+                    if(matchingModelsReplacementIssue)
+                    {
+                        System.Text.StringBuilder issueMsg = new System.Text.StringBuilder(Environment.NewLine + Environment.NewLine);
+                        foreach(DeltaAPIModel deltaMod in repeatingMatchingModels)
+                        {
+                            apiCount++;
+                            issueMsg.Append(apiCount + "). " + deltaMod.MatchingAPIName + Environment.NewLine);
+                        }
+                        Reporter.ToUser(eUserMsgKey.MultipleMatchingAPI, "Below Matching API Models selected to be replaced/overwritten multiple times:" + issueMsg.ToString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There's no need to call 'ToString()', the compiler will do it for you.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4489955)